csvqb/csvqb/models/cube/csvqb/components/attribute.py

```diff
@@ -92,3 +145,17 @@ def from_data(
 
     def validate_data(self, data: PandasDataTypes) -> List[ValidationError]:
         return []  # TODO: implement this
+
+
+@dataclass
+class NewQbAttributeLiteral(NewQbAttribute, QbAttributeLiteral):
+    new_attribute_values: List[NewQbAttributeValue] = field(
```

Comment:
Since this is a `literal` you shouldn't need any `NewQbAttributeValue`s here - and it should be inherited from `NewQbAttribute` anyway.

We could add a pydantic validation error if the user populates these values on a literal, telling them they shouldn't be doing that.

Reply:
Can we talk about removing this? Since getting rid of it causes problems.